movies/1990/edward-scissorhands.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Edward Scissorhands ",
+  "year": 1990,
+  "runtime": 105,
+  "categories": [
+    "drama",
+    "fantasy",
+    "romance"
+  ],
+
+  "description" : "An artificial man, who was incompletely constructed and has scissors for hands, leads a solitary life. Then one day, a suburban lady meets him and introduces him to her world.",
```

Comment:
This should not be here.  Please follow the format provided here > https://github.com/mikeleguedes/json-movie-list